sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/metadata/TableMetaDataLoader.java

```diff
@@ -54,23 +54,23 @@
  */
 @RequiredArgsConstructor
 public final class TableMetaDataLoader {
-    
+
```

Comment:
Please do not change code format if unnecessary. Can you revert format changes for this class?